### DISCUSSION TITLE
Censoring rate influence on results

### DISCUSSION THREAD

**Discussion by tomas-silveira** [sebp/scikit-survival#267] [Q&A]
Hello, this is more of a theoretical question, but I was wondering if censoring rates have a big influence on the quality of the results from these survival models. In my case, I'll be studying the applicability of survival analysis for churn prediction, which uses a dataset with a  very large number of non-churners. What would be your recommendation for this case? For now, we'll be experiment with undersampling and oversampling techniques.

**Comment by sebp** [comment]
It probably does, both for the model itself but also the evaluation metric. Regarding the latter, the [user guide covers this to some extent](https://scikit-survival.readthedocs.io/en/latest/user_guide/evaluating-survival-models.html#Simulation-Study).
